[2023-04-09T00:37:58.261+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: api_dag.is_api_available manual__2023-04-09T00:37:56.121778+00:00 [queued]>
[2023-04-09T00:37:58.267+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: api_dag.is_api_available manual__2023-04-09T00:37:56.121778+00:00 [queued]>
[2023-04-09T00:37:58.268+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2023-04-09T00:37:58.269+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 2
[2023-04-09T00:37:58.270+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2023-04-09T00:37:58.278+0000] {taskinstance.py:1383} INFO - Executing <Task(HttpSensor): is_api_available> on 2023-04-09 00:37:56.121778+00:00
[2023-04-09T00:37:58.283+0000] {standard_task_runner.py:55} INFO - Started process 9969 to run task
[2023-04-09T00:37:58.287+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'api_dag', 'is_api_available', 'manual__2023-04-09T00:37:56.121778+00:00', '--job-id', '77', '--raw', '--subdir', 'DAGS_FOLDER/api.py', '--cfg-path', '/tmp/tmpncjedywc']
[2023-04-09T00:37:58.290+0000] {standard_task_runner.py:83} INFO - Job 77: Subtask is_api_available
[2023-04-09T00:37:58.348+0000] {task_command.py:376} INFO - Running <TaskInstance: api_dag.is_api_available manual__2023-04-09T00:37:56.121778+00:00 [running]> on host c12a518d534c
[2023-04-09T00:37:58.398+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=api_dag
AIRFLOW_CTX_TASK_ID=is_api_available
AIRFLOW_CTX_EXECUTION_DATE=2023-04-09T00:37:56.121778+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-04-09T00:37:56.121778+00:00
[2023-04-09T00:37:58.400+0000] {http.py:120} INFO - Poking: api/
[2023-04-09T00:37:58.410+0000] {base.py:71} INFO - Using connection ID 'user_api' for task execution.
[2023-04-09T00:37:58.412+0000] {http.py:148} INFO - Sending 'GET' to url: https://randomuser.me/api/
[2023-04-09T00:37:58.643+0000] {base.py:213} INFO - Success criteria met. Exiting.
[2023-04-09T00:37:58.662+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=api_dag, task_id=is_api_available, execution_date=20230409T003756, start_date=20230409T003758, end_date=20230409T003758
[2023-04-09T00:37:58.719+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2023-04-09T00:37:58.746+0000] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
